### ISSUE TITLE
SideNav is a misnomer

### ISSUE THREAD

**Issue by chriscurnow** [angular/material2#5160]
####Comment:

This is perhaps really a comment about Material in general than Angular Material. I'm not sure as my only experience with Material is with Angular (2+).

It has taken me a long time to work out that SideNav does not provide any navigation at all – I'd love to be proven wrong on that but I can't find any reference to actual navigation.

SideNav should really SideBar – you can put any content you like in it.

At the same time, I can't find anything in Angular Material that assists in creating a vertical nav list in Angular. From what I read, Material pioneered side navs as we see in just about every web app out there now (whether provided by Material or not) but there is no easy way to create one with Angular Material.

I hope I'm wrong about all this and very much welcome being corrected.

**Comment by willshowell** [comment]
It should be as simple as putting a nav-list inside the sidenav

https://material.angular.io/components/component/list#navigation-lists

**Comment by kara** [comment]
Please keep GitHub issues for bug reports / feature requests. Better avenues for troubleshooting / questions are stack overflow, gitter, mailing list, etc.  See [readme](https://github.com/angular/material2/blob/master/CONTRIBUTING.md#-got-a-question-or-problem) for more info.

**Comment by chriscurnow** [comment]
This is not a troubleshooting issue. I reported it here because it is effectively a "bug" in the naming convention. SideNav is NOT a navigation tool. It is simply a sidebar in which you can put anything you want. So I stand by my comment that is misnamed and therefore confusing.

I'm not one for complaining about things. FWIW, I think Angular Material is absolutely brilliant. I love it.

For mine, SideNav is the only area in which it falls down – and that is because it raises expectations it is not meant to meet.

I do know how to create a navigation list and have spent many hours trying to create one that behaves like the SideNavs we see all over Google. Take the side nav on the Docs page at angular.io.

I'd love to be able to create that with Angular Material but instead to get anything close, I have had to use Bootstrap.

I can't even find anyone who has done it.

In summary then my comment here is not about asking for help to create a sidenav, rather about the behaviour of sidenav. 

I also realise that all the documentation is far too far down the track for this name to be changed. But some explanation or discussion would be helpful.

It is strange however that it is Material that made the now ubiquitous sideNav famous but now seems to eschew it.
